src/Integrations/Integrations/CakePHP/CakePHPIntegration.PHP5.php

```diff
@@ -58,13 +58,18 @@ public function init()
                 $integration->rootSpan->meta[Tag::SPAN_KIND] = 'server';
             }
 
+            $integration->rootSpan->meta[Tag::SPAN_KIND] = 'server';
+            $integration->rootSpan->meta[Tag::COMPONENT] = $this->getName();
```

Comment:
I would use the `NAME` constant instead of the method in each case. It will have lower overhead, and I'm not really sure why there is a `getName` method, maybe some artifact of historical compatibility with OpenTelemetry or something.